spring-5-reactive-client/src/test/java/com/baeldung/reactive/simultaneouswebclient/ClientIntegrationTest.java

```diff
@@ -0,0 +1,71 @@
+package com.baeldung.reactive.simultaneouswebclient;
+
+import org.junit.Test;
+import org.junit.Before;
+import org.junit.After;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import com.github.tomakehurst.wiremock.WireMockServer;
+
+import java.util.Arrays;
```

Comment:
Lets' remove this unused import please and again update the package as mentioned before.

Reply:
fixed